noxfile.py

```diff
@@ -28,6 +28,7 @@
 
 #: Cirrus-CI environment variable hook.
 PY_VER = os.environ.get("PY_VER", _PY_VERSIONS_ALL)
+COVERAGE = os.environ.get("COVERAGE", False)
```

Comment:
We have discussed offline using [Nox `posargs`](https://nox.thea.codes/en/stable/config.html#passing-arguments-into-sessions) (also available with Tox, when we get there) rather than an environment variable.

Reply:
@lbdreyer I've seen bea9a6837d4adabd60ba6ae10a8257c7daa41517, but the What's New entry also needs a minor change too

Reply:
Good point! I will update that too!